glean-core/README.md

```diff
@@ -0,0 +1,50 @@
+# Glean SDK
+
+The `Glean SDK` is a modern approach for a Telemetry library and is part of the [Glean project](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
+
+## `glean-core`
+
+This library provides the core functionality of the Glean SDK, including implementations of all metric types, the ping serializer and the storage layer.
+It's used in all platform-specific wrappers.
```

Comment:
Can we call out that this is not meant to be used directly by users? This is meant to be consumed by other Glean packages?

Reply:
Well, it can :D